src/ui/public/partials/info.html

```diff
@@ -5,7 +5,7 @@
   tooltip="{{info}}"
   tooltip-placement="{{placement}}"
   tooltip-popup-delay="250"
-  tooltip-trigger="focus"
+  tooltip-trigger="focus click"
```

Comment:
The default behavior of the tooltip is to show on `mounseenter`, which is in line with how our new [EuiToolTip](https://elastic.github.io/eui/#/display/tooltip) behaves. Can we just delete this line to enable this default behavior? Then we also won't need the changes in `tooltip.js`, above.

Reply:
Done